distribution/docs/src/main/resources/_contents/_sources/catalog-providers-contents.adoc

```diff
@@ -1,8 +1,91 @@
+{branding} uses Solr for data storage. Whenever the {branding} Solr ingests data, it creates an index for it in the data directory.
```

Comment:
✏️  The information in this paragraph and the next apply only to the embedded and http/external, not Solr, so we may want to move those after the list of providers and specify that it does not apply to Solr Cloud.